gui/data_loc_info.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, parent: "ControlMain"):
         self.hBoxDPathParams2.addWidget(self.prefix_ledit)
         self.dataNumstartLabel = QtWidgets.QLabel("File Number Start:")
         self.file_numstart_ledit = QtWidgets.QLineEdit()
+        self.file_numstart_ledit.setText("1")
```

Comment:
Let's create a Jira ticket and/or pull request to talk with beamline scientists to remove file numbers entirely.